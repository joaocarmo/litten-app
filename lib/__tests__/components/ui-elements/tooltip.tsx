import TestRenderer from 'react-test-renderer'
import ThemeProvider from '@components/theme/provider'
import { UITooltip } from '@ui-elements'

describe('Snapshot test for the "UITooltip" component', () => {
  it('renders correctly', () => {
    const element = TestRenderer.create(
      <ThemeProvider>
        <UITooltip>Some text here</UITooltip>
      </ThemeProvider>,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })
})
