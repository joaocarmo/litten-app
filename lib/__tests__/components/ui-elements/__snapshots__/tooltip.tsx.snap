// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the "UITooltip" component renders correctly 1`] = `
<View
  collapsable={false}
  style={
    {
      "opacity": 0,
      "position": "absolute",
      "top": 0,
      "zIndex": 1,
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      [
        {
          "elevation": 6,
          "shadowColor": "#000000",
          "shadowOffset": {
            "height": 4,
            "width": 0,
          },
          "shadowOpacity": 0.1,
          "shadowRadius": 2,
        },
        {
          "backgroundColor": "#84bff1",
          "borderRadius": 16,
          "paddingBottom": 6,
          "paddingLeft": 14,
          "paddingRight": 14,
          "paddingTop": 6,
        },
        undefined,
      ]
    }
  >
    <Text
      style={
        {
          "color": "#ffffff",
          "fontWeight": "700",
        }
      }
    >
      Some text here
    </Text>
  </View>
</View>
`;
